client/lib/api/who_service.dart

```diff
@@ -4,7 +4,8 @@ import 'package:who_app/main.dart';
 import 'package:who_app/api/user_preferences.dart';
 import 'package:http/http.dart' as http;
 import 'dart:io' as io;
-
+import 'package:get_proxy/get_proxy.dart';
```

Comment:
Instead of `get_proxy`, should it be `http_proxy`? You're not getting a proxy but using it for requests. It should match the `http.post` API as much as possible. The used class can then be `httpProxy` which more closely matches.

Reply:
Just a heads up that get_proxy was literally just getting the proxy information.  But I'll move the http.post alternative into this package and rename it as you suggest @brunobowden .